### ISSUE TITLE
[ALSOFT] (EE) Failed to set real-time priority for thread: Operation not permitted (1)

### ISSUE THREAD

**Issue by tejohnso** [SFML/SFML#1832]
## Subject of the issue

Nag / Warning during startup that was not there previously. Likely changed during a system update in the last year or so.

## Your environment

* Linux 5.14.11-arch1-1 x86_64 GNU/Linux
* SFML 2.5.1
* GCC 11.1.0
* no special compiler flags

## Steps to reproduce

```cpp
#include <SFML/Audio.hpp>
#include <thread>
#include <chrono>

int main() {
    sf::SoundBuffer buffer;
    if (!buffer.loadFromFile("gulp.ogg")) return -1;

    sf::Sound sound;
    sound.setBuffer(buffer);
    sound.play();
    std::this_thread::sleep_for(std::chrono::seconds(1));
    return 0;
}

```
Compile and run with no special flags

## Expected behavior

Sound plays, exit code 0, no stdout output

## Actual behavior

Sound plays, exit code 0, stdout output:

> [ALSOFT] (EE) Failed to set real-time priority for thread: Operation not permitted (1)
> [ALSOFT] (EE) Failed to set real-time priority for thread: Operation not permitted (1)

**Comment by eXpl0it3r** [comment]
Looking a bit around, this seems much more a system configuration issue, where OpenAL-soft doesn't get real-time priority, which can impact the audio playback. 

See also:
- https://github.com/kcat/openal-soft/issues/554
- https://github.com/telegramdesktop/tdesktop/issues/17025#issuecomment-934255022

**Comment by ForeverZer0** [comment]
As noted, this is a system configuration issue (also an Arch user who had the same problem). I fixed it with the following method:

1. Open/create the following config file: `/etc/security/limits.d/audio.conf`
2. Add the following lines:
```
@audio   -  rtprio     95
@audio   -  memlock    unlimited
```
3. Add yourself to the user group `audio`, create it if it does not exist:
```
groupadd audio
usermod -a -G audio <username>
```
4. Logout or reboot

This worked for me on Arch, 5.17.3 kernel, with KDE/Plasma desktop.

**Comment by ChrisThrasher** [comment]
Closed due to #2749 removing OpenAL entirely
